tests/integration/conftest.py

```diff
@@ -1,44 +1,111 @@
+"""pytest configuration for integration tests."""
+import argparse
 import pytest
-from tempfile import TemporaryDirectory
-from sbws.util.parser import create_parser
-from sbws.util.config import get_config
-from sbws.util.stem import launch_tor
-import sbws.core.init
 import os
 
+from sbws.core import init
+from sbws.lib.circuitbuilder import GapsCircuitBuilder as CB
+from sbws.lib.destination import DestinationList
+from sbws.lib.relaylist import RelayList
+from sbws.util.config import _get_default_config
+from sbws.util.stem import launch_tor
+
+
+class _PseudoArguments(argparse.Namespace):
```

Comment:
I think we should add a function to this class that adds arbitrary attributes after the object has been created.

For example, when we want to test `sbws.core.stats` and call its `main()`, we can do the following in order to be able to avoid the `getattr()` and go back to `args.error_types`.

```
args.add_args(error_types=False)
```

Reply:
hmm, i could add:
```
    def add_arg(self, arg):
        setattr(self, arg)
```
but then why just not using ``setattr`` 

In the tests i could just initialize the args with:
```
    args = _PseudoArguments(directory=sbwshome_dir, output=sbwshome_dir,
                            scale=False, log_level='debug', error_types=False)
```
no need for ``add_arg``
In non-tests, you ensure all the possible args are present with ``config.py``